src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,15 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        ColorSupplier supplier = new ColorSupplier();
+        for(int i = 0; i < 3; i++) {
+            Colors value = supplier.getRandomColor();
+            System.out.println(value);
+        }
+        Lottery lottery = new Lottery();
+        for(int i = 0; i < 3; i++) {
+            int number = Lottery.getRandomBall();
```

Comment:
Avoid using static methods as per the checklist. It's recommended to create an instance of the Lottery class and use its methods. You have already created an instance of Lottery on line 10, so you should use that instance instead of calling a static method.